### PR TITLE
 Balearic Islands Carbon production capacity

### PR DESCRIPTION
**Pull request by blackleg**
- Added Balearic Islands Carbon production capacity
- Added El Hierro Island Storage capacity

### CODE REVIEW

**Comment by blackleg** [comment]
@corradio Updated

**Comment by jarek** [comment]
The reason for https://github.com/tmrowco/electricitymap/pull/637/commits/83bbf70a692bf2bb87b84491d9a49381c1f27c7b having such a huge diff (everything red and green, +234 −230) is because the line ending format has been changed.

Other than the line endings, the change was:

     - Spain: [ENTSO-E](https://transparency.entsoe.eu/generation/r2/installedGenerationCapacityAggregation/show?name=&defaultValue=false&viewType=TABLE&areaType=BZN&atch=false&dateTime.dateTime=01.01.2017+00:00|UTC|YEAR&dateTime.endDateTime=01.01.2017+00:00|UTC|YEAR&area.values=CTY|10YES-REE------0!BZN|10YES-REE------0&productionType.values=B01&productionType.values=B02&productionType.values=B03&productionType.values=B04&productionType.values=B05&productionType.values=B06&productionType.values=B07&productionType.values=B08&productionType.values=B09&productionType.values=B10&productionType.values=B11&productionType.values=B12&productionType.values=B13&productionType.values=B14&productionType.values=B20&productionType.values=B15&productionType.values=B16&productionType.values=B17&productionType.values=B18&productionType.values=B19)
    +- Spain (Canary Islands): ^M
    +  - Hydraulic Storage [goronadelviento.es](http://www.goronadelviento.es/)^M
    +- Spain (Balearic Islands): ^M
    +  - Carbon: [wikipedia.org](https://es.wikipedia.org/wiki/Central_t%C3%A9rmica_de_Es_Murterar)^M
     - Sweden: [ENTSO-E](https://transparency.entsoe.eu/generation/r2/installedGenerationCapacityAggregation/show)

That means the following was added:

    - Spain (Canary Islands): ^M
      - Hydraulic Storage [goronadelviento.es](http://www.goronadelviento.es/)^M
    - Spain (Balearic Islands): ^M
      - Carbon: [wikipedia.org](https://es.wikipedia.org/wiki/Central_t%C3%A9rmica_de_Es_Murterar)^M

Oli, if you fix the line endings so that https://github.com/tmrowco/electricitymap/pull/637/files doesn't show the entire README in red/green, then squash the commits when merging it should be fine in history.

File: README.md [hunk 1]
```diff
@@ -1,230 +1,234 @@
-# electricitymap [![Slack Status](http://slack.tmrow.co/badge.svg)](http://slack.tmrow.co)
```
Comment (corradio) [inline README.md]: Hmm what happened here? How come the whole file was modified?
Can you try to fix that, or maybe revert and only modify the relevant parts?
Thanks!

**Comment by corradio** [comment]
Good point, let me do that.

File: README.md [hunk 2]
```diff
@@ -143,6 +143,10 @@ Production capacities are centralized in the [capacities.json](https://github.co
...
 - Slovenia: [ENTSO-E](https://transparency.entsoe.eu/generation/r2/installedGenerationCapacityAggregation/show)
 - Spain: [ENTSO-E](https://transparency.entsoe.eu/generation/r2/installedGenerationCapacityAggregation/show?name=&defaultValue=false&viewType=TABLE&areaType=BZN&atch=false&dateTime.dateTime=01.01.2017+00:00|UTC|YEAR&dateTime.endDateTime=01.01.2017+00:00|UTC|YEAR&area.values=CTY|10YES-REE------0!BZN|10YES-REE------0&productionType.values=B01&productionType.values=B02&productionType.values=B03&productionType.values=B04&productionType.values=B05&productionType.values=B06&productionType.values=B07&productionType.values=B08&productionType.values=B09&productionType.values=B10&productionType.values=B11&productionType.values=B12&productionType.values=B13&productionType.values=B14&productionType.values=B20&productionType.values=B15&productionType.values=B16&productionType.values=B17&productionType.values=B18&productionType.values=B19)
+- Spain (Canary Islands): 
+  - Hydraulic Storage [goronadelviento.es](http://www.goronadelviento.es/)
+- Spain (Balearic Islands): 
+  - Carbon: [wikipedia.org](https://es.wikipedia.org/wiki/Central_t%C3%A9rmica_de_Es_Murterar)
```
Comment (corradio) [inline README.md]: I think you mean coal?

File: README.md [hunk 3]
```diff
@@ -143,6 +143,10 @@ Production capacities are centralized in the [capacities.json](https://github.co
 - Slovakia: [SEPS](https://www.sepsas.sk/Dokumenty/RocenkySed/ROCENKA_SED_2015.pdf)
 - Slovenia: [ENTSO-E](https://transparency.entsoe.eu/generation/r2/installedGenerationCapacityAggregation/show)
 - Spain: [ENTSO-E](https://transparency.entsoe.eu/generation/r2/installedGenerationCapacityAggregation/show?name=&defaultValue=false&viewType=TABLE&areaType=BZN&atch=false&dateTime.dateTime=01.01.2017+00:00|UTC|YEAR&dateTime.endDateTime=01.01.2017+00:00|UTC|YEAR&area.values=CTY|10YES-REE------0!BZN|10YES-REE------0&productionType.values=B01&productionType.values=B02&productionType.values=B03&productionType.values=B04&productionType.values=B05&productionType.values=B06&productionType.values=B07&productionType.values=B08&productionType.values=B09&productionType.values=B10&productionType.values=B11&productionType.values=B12&productionType.values=B13&productionType.values=B14&productionType.values=B20&productionType.values=B15&productionType.values=B16&productionType.values=B17&productionType.values=B18&productionType.values=B19)
+- Spain (Canary Islands): 
+  - Hydraulic Storage [goronadelviento.es](http://www.goronadelviento.es/)
```
Comment (corradio) [inline README.md]: In order to stay consistent with the way we name things, can you replace with `Hydro storage`?

**Comment by blackleg** [comment]
Fixed

**Pull request merged**